Resources/views/line/video.html.twig

```diff
@@ -0,0 +1,27 @@
+{% set videoType = content.getField( 'file' ).value.mimeType|default( 'video/mp4' ) %}
+
```

Comment:
Content may contain lot of video files, we should not declare this settings every time. It should be added only once on every page (btw, this setting was missing in legacy too).